clients/sled-agent-client/src/lib.rs

```diff
@@ -25,6 +25,9 @@ progenitor::generate_api!(
     //TODO trade the manual transformations later in this file for the
     //     replace directives below?
```

Comment:
it seems like there are still quite a few of them, so I assume this comment is no less applicable than before your change; is that right?

Reply:
That's right.  Much of what's left are stuff I wouldn't consider "primitives" (e.g., InstanceRuntimeState).  There's some net stuff that could get the same treatment but I wasn't sure so I left it alone.